frameworks/digital-outcomes-and-specialists-2/questions/briefs/locationAllLots.yml

```diff
@@ -0,0 +1,20 @@
+name: Location
+question: Location
+id: location
+type: radios
+options:
+  - label: Scotland
+  - label: North East England
+  - label: North West England
+  - label: Yorkshire and the Humber
+  - label: East Midlands
+  - label: West Midlands
+  - label: East of England
+  - label: Wales
+  - label: London
+  - label: South East England
+  - label: South West England
+  - label: Northern Ireland
+  - label: International (outside the UK)
```

Comment:
Totally struggling to imagine what this even means in the context of DOS, but eh

Reply:
I think user research participants can be international. If that's ever actually happened, I don't know.

Reply:
Participants wanted in Hawaii. No particular reason... :-D

Reply:
It would be good to have values against the options as well, is this something that is possible?

The reason why I say this is because these labels have spaces and when we send virtual page views, we use this information to construct URLs (not very friendly when the URLs contain spaces)

Reply:
We have to go with whatever's stored in the API, so this really has to match "location.yml" file. I agree it's a pity we don't automatically construct a prettier value for this things, but that'd involve some quite complex changes across the content loader and the apps :(

Reply:
(Hmm actually maybe we strip spaces anyway, so maybe it's just a question of changing the frontend to remove the spaces in _its_ URLs. Either way, no change required here.)

Reply:
Don't we slugify things like this when creating URLs?

Reply:
I don't think we use these values in URL's anywhere, but they might be part of the URL's for the virtual page views. I'm not very familiar with them to be honest. @DilwoarH ?

Reply:
They would be in the URLs for when we search by location. But if we don't like the look of those, that's a buyer-app fix, not a change here, I think.

Reply:
yeah agreed with you @galund 

Thanks for addressing the comment.